scripts/storage-emulator-integration/import/windows-emulator-data/storage_export/metadata/fake-project-id.appspot.com%5Ctest_upload.jpg.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "test_upload.jpg",
```

Comment:
can you include a test with a nested directory structure

Reply:
Done.